### PR TITLE
Manage PKI renew on restore

### PR DESCRIPTION
**Pull request by tiraboschi**

### CODE REVIEW

**Comment by tiraboschi** [comment]
Fixes: https://bugzilla.redhat.com/1686445

**Comment by didib** [comment]
Looks good to me.

File: defaults/main.yml [hunk 1]
```diff
@@ -55,6 +55,7 @@ he_enable_libgfapi: false
 he_enable_hc_gluster_service: false
 he_vm_mac_addr: null
 he_remove_appliance_rpm: true
+he_pki_renew_on_restore: false
```
Comment (irosenzw) [inline defaults/main.yml]: Add it to the README.md file please

Comment (tiraboschi) [reply defaults/main.yml]: Done

**Pull request merged**